server/engine/src/test/java/org/eclipse/lsp/cobol/dialects/ibm/CobolLineIndicatorProcessorImplTest.java

```diff
@@ -22,7 +22,6 @@
 import com.google.common.collect.ImmutableList;
 import java.util.List;
 import java.util.Optional;
-import lombok.val;
```

Comment:
This feature was not automatically delomboked, so I decided to remove it

Reply:
Which version of Java is actually being targeted?
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/0d28f1bf5cc49cb956ad21e5fa8cd28650671c66/server/pom.xml#L23-L24
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/0d28f1bf5cc49cb956ad21e5fa8cd28650671c66/.github/workflows/build.yml#L90

Asking for a friend - https://openjdk.org/jeps/286

Reply:
Java 8, but you can use Jdk 11 to compile it